bms/templates/registration/bug_view.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+</head>
+<body>
+<h4>
+	Bug view
+
+	<a href="/logout"><span class="glyphicon glyphicon-log-out pull-right"></span>Log Out </a>
+</h4>
+</body>
+</html>
```

Comment:
what is purpose of this file, why you have added html, head, title tags here.
Header, navigation and footer sections should be in a common file, that should not repeat in very templates. Need to create new task in next sprint.